### ISSUE TITLE
Only US number format is allowed.

### ISSUE THREAD

**Issue by angelcervera** [sourcerer-io/sourcerer-app#481]
In https://sourcerer.io/settings#jobs is not possible to use the phone number from outside of the US.

For example, my phone number in Ireland is +353861245678 with international prefix or 0861245678 without international prefix.

To reproduce the problem, try to set this number and save.

**Comment by herodrigues** [comment]
Sadly, this is true. The input mask only works for US numbers.

**Comment by yandzee** [comment]
Yes, we only support phone numbers in international format, `+353861245678` works.

**Comment by anatolystansler** [comment]
@yandzee Phone field works really strange for me. Max length bound counts plus in first case and not in second. It is definitely hard to type sample phone number to the edit.
![1](https://user-images.githubusercontent.com/29913247/54632852-0324cc00-4a90-11e9-8449-3e11e454d2b5.gif)
![2](https://user-images.githubusercontent.com/29913247/54632854-0324cc00-4a90-11e9-9eeb-15d09a7a917d.gif)
